h2o-py/h2o/automl/_estimator.py

```diff
@@ -404,11 +411,45 @@ def __validate_algo_parameters(self, algo_parameters):
     def __validate_frame(self, fr, name=None, required=False):
         return H2OFrame._validate(fr, name, required=required)
 
+    def __validate_distribution(self, distribution):
+        if is_type(distribution, str):
+            distribution = distribution.lower()
+            if distribution == "custom":
+                raise H2OValueError('Distribution "custom" has to be specified as a '
+                                    'dictionary with their respective parameters, e.g., '
+                                    '`dict(distribution = \"custom\", custom_distribution_func = \"...\"))`.')
+            return distribution
+        if is_type(distribution, dict):
+            dist = distribution["distribution"].lower()
+            ALLOWED_DISTRIBUTION_PARAMETERS = dict(
```

Comment:
why the upper case? isn't your Py linter complaining? it's just a normal local variable, but apparently an arrogant one :)

Reply:
My Py linter is complaining about a lot of things including this one. It's lower-cased now.